src/extension/gdal/src/main/java/org/geoserver/gdal/GDALStatus.java

```diff
@@ -68,9 +68,13 @@ public boolean isEnabled() {
         return Optional.ofNullable("");
     }
 
-    public String getGDALWrapperJarVersion() {
+    public Optional<String> getGDALWrapperJarVersion() {
         if (isAvailable()) {
-            return GeoTools.getVersion(gdal.class).toString();
+            Version v = GeoTools.getVersion(gdal.class);
+            if (v == null) {
+                return Optional.empty();
```

Comment:
So when does this occur? Probably when running from a development environment? I kind of thought you would use the GDAL bindings if available and report back the version of GDAL used.